docs/integrations/sources/mssql.md

```diff
@@ -306,8 +306,9 @@ If you do not see a type in this list, assume that it is coerced into a string.
 
 | Version | Date       | Pull Request | Subject                                                                                                |
 |:--------|:-----------| :----------------------------------------------------- |:-------------------------------------------------------------------------------------------------------|
-| 0.4.11   | 2022-07-22 | [14714](https://github.com/airbytehq/airbyte/pull/14714) | Clarified error message when invalid cursor column selected |
-| 0.4.10   | 2022-07-14 | [14574](https://github.com/airbytehq/airbyte/pull/14574) | Removed additionalProperties:false from JDBC source connectors |
+| 0.4.12  | 2022-08-03 | [14910](https://github.com/airbytehq/airbyte/pull/14910) | Standardize spec for DB connectors that support CDC replication |
```

Comment:
Maybe add more specific description about what exactly is standardized. Something like:

```suggestion
| 0.4.12  | 2022-08-03 | [14910](https://github.com/airbytehq/airbyte/pull/14910) | Standardize spec for CDC replication. Rename the `replication` field to `replication_method`. |
```

Reply:
@tuliren updated changelog description